e2e/runner/bitcoin.go

```diff
@@ -286,6 +296,57 @@ func (r *E2ERunner) SendToTSSFromDeployerWithMemo(
 	return txid, nil
 }
 
+func (r *E2ERunner) InscribeToTSSFromDeployerWithMemo(
+	amount float64,
+	inputUTXOs []btcjson.ListUnspentResult,
+	memo []byte,
+) (*chainhash.Hash, error) {
+	builder := InscriptionBuilder{sidecarUrl: "http://bitcoin-node-sidecar:8000", client: http.Client{}}
+
+	address, err := builder.GenerateCommitAddress(memo)
+	if err != nil {
+		return nil, err
+	}
+	r.Logger.Print("received inscription commit address %s", address)
+
+	receiver, err := chains.DecodeBtcAddress(address, r.GetBitcoinChainID())
+	if err != nil {
+		return nil, err
+	}
+
+	txnHash, err := r.sendToAddrFromDeployerWithMemo(amount, receiver, inputUTXOs, []byte(constant.DonationMessage))
```

Comment:
It feels to me `constant.DonationMessage` is not a required piece for the commit tx. Can we have comments on the goal of this call (what needs to be done here in minimum)?